_posts/2019-08-01-CocoaPods-1.8.0-beta.markdown

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title:  "CocoaPods 1.8 Beta is Here!"
+author: dnkoutso
+categories: cocoapods releases
+---
+
+_CocoaPods 1.8_ switches to CDN as the default and comes with a few enhancements!
+
+<!-- more -->
+
+This is an exciting release as the [master specs repo](https://github.com/CocoaPods/Specs) is no longer required to be cloned to start using CocoaPods!
+
+### CDN as Default
+
+CDN support was first introduced in the [1.7 release](http://blog.cocoapods.org/CocoaPods-1.7.0-beta/) and aims to speed up initial setup and dependency analysis dramatically. With 1.8, CocoaPods no longer requires to clone the [master specs repo](https://github.com/CocoaPods/Specs) in order to function which has become huge since the inception of the project.
+
+You may safely delete the [master specs repo](https://github.com/CocoaPods/Specs) with the following command:
+
+```
+pod repo remove master
+```
+
+For more information regarding CDN, please read our previous blog post [here](http://blog.cocoapods.org/CocoaPods-1.7.2/)!
+
+### `info_plist` Podspec DSL
+
+TODO
```

Comment:
@amorde can you provide a paragraph and any screenshots for this? I can write something up instead.